common-navigation/src/main/java/com/gevondov/common/navigation/CommonFragmentFactory.kt

```diff
@@ -0,0 +1,15 @@
+package com.gevondov.common.navigation
+
+import androidx.fragment.app.FragmentFactory
+import com.gevondov.feature.countries.view.CountriesViewImpl
+import org.koin.core.KoinComponent
+import org.koin.core.get
+
+class CommonFragmentFactory : FragmentFactory(), KoinComponent {
```

Comment:
Why there are two CommonFragmentFactory? First one is in `common-containers` module, and the second one is here in `common-navigation`  module.

Reply:
First one is ContainerFragmentFactory